docs/2022/ui/updates/ayush/2022-06-02.md

```diff
@@ -55,4 +55,100 @@ GSOC 2022 weekly update.
 ### Conclusion and Further Plans
 
 - Compatible meeting timing was decided
-- Need to keep gsoc page updated
\ No newline at end of file
+- Need to keep gsoc page updated
+
+---
+title: GSOC 2022 weekly update.
+author: Ayush Kumar
+author_url: https://github.com/Ayush7614
+author_image_url: https://avatars.githubusercontent.com/u/67006255?v=4
+tags: [gsoc22, react, ui, api]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2022 Ayush Kumar <ayushknj3@gmail.com>
+-->
+
+## Meeting 2
+*(June 9th, 2022)*
+
+GSOC 2022 weekly update.
+
+<!--truncate-->
+
+### Attendees
+
+- [Anupam Ghosh](https://github.com/ag4ums)
+- [Gaurav Mishra](https://github.com/GMishx)
+- [Shaheem Azmal M MD](https://github.com/shaheemazmalmmd)
+- [Kaushlendra Pratap](https://github.com/Kaushl2208)
+- [Ayush Bhardwaj](https://github.com/hastagAB)
+- [Vivek Kumar](https://github.com/viv9k)
+- [Avinal Kumar](https://github.com/avinal)
+- [Sushant Kumar](https://github.com/its-sushant)
+- [Rohit Pandey](https://github.com/rohitpandey49)
+- [Thanvi Lahari Pendyala](https://github.com/Pendyala-thanvi)
+- [Ayush Kumar](https://github.com/Ayush7614)
+- [Krishna Mahato](https://github.com/krishna9304)
+- [Soham Banerjee](https://github.com/soham4abc)
+
+### Discussions
+
+- Fossology Setup Done
+- My Sem end exams are going on, after which I start to contribution.
+- Exploring Code, APIs
+- Discussion with mentors
+
+###  Conclusion and Further Plans
+
+- Watch Fossology Tutorials on fossology youtube channel for better understanding of the project.
+
+---
+title: GSOC 2022 weekly update.
+author: Ayush Kumar
+author_url: https://github.com/Ayush7614
+author_image_url: https://avatars.githubusercontent.com/u/67006255?v=4
+tags: [gsoc22, react, ui, api]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2022 Ayush Kumar <ayushknj3@gmail.com>
+-->
+
+## Meeting 3
+*(June 16th, 2022)*
```

Comment:
Hello @Ayush7614 ,
As from June 13, the coding period started, so please move updates to a new weekly file under same folder as done in previous run:
https://github.com/fossology/gsoc/tree/main/docs/2021/copyrights/updates

Reply:
Any updates on this @Ayush7614 ?